TODO

```diff
@@ -0,0 +1,60 @@
+# Backlog
```

Comment:
thanks for merging, did you intent to commit the TODO file?

Reply:
no, sigh - i had just renamed it and updated my .gitignore on main but accidentally committed it in the pr.  i've cleaned it up now, though it will live on forever in the git history.  there's nothing sensitive in there